x-pack/plugins/snapshot_restore/public/application/sections/policy_edit/policy_edit.tsx

```diff
@@ -106,21 +106,39 @@ export const PolicyEdit: React.FunctionComponent<RouteComponentProps<MatchParams
     history.push(encodeURI(`${BASE_PATH}/policies/${encodeURIComponent(name)}`));
   };
 
+  const renderSaveError = () => {
+    return saveError ? (
+      <SectionError
+        title={
+          <FormattedMessage
+            id="xpack.snapshotRestore.editPolicy.savingPolicyErrorTitle"
+            defaultMessage="Cannot save policy"
+          />
+        }
+        error={saveError}
+      />
+    ) : null;
+  };
+
+  const clearSaveError = () => {
+    setSaveError(null);
+  };
+
   const renderLoading = () => {
-    return errorLoadingPolicy ? (
-      <SectionLoading>
+    return isLoadingPolicy ? (
```

Comment:
I don't think this was behaving correctly before. I don't see why we would want to check `errorLoadingPolicy` to determine whether we should render a loading state or not.